tests/onegov/town6/test_views_homepage.py

```diff
@@ -88,3 +88,23 @@ def test_homepage(client):
     client.app.org.disable_chat = True
     transaction.commit()
     assert not client.get('/').pyquery('.chat')
+
+
```

Comment:
Do we need two duplicate tests? I think one can be removed, since `town6` uses the `org` implementation.